src/frontend/src/pages/ChangeRequestDetailPage/ImplementedChangesList.tsx

```diff
@@ -4,44 +4,48 @@
  */
 
 import { ImplementedChange } from 'shared';
-import { datePipe, emDashPipe, fullNamePipe, wbsPipe } from '../../utils/pipes';
+import { datePipe, fullNamePipe, wbsPipe } from '../../utils/pipes';
 import { routes } from '../../utils/routes';
 import { Link, ListItem, List, Typography } from '@mui/material';
-import PageBlock from '../../layouts/PageBlock';
 import { Link as RouterLink } from 'react-router-dom';
 import DynamicTooltip from '../../components/DynamicTooltip';
+import InfoBlock from '../../layouts/InfoBlock';
 
 interface ImplementedChangesListProps {
   changes: ImplementedChange[];
   overallDateImplemented?: Date;
 }
 
-const ImplementedChangesList: React.FC<ImplementedChangesListProps> = ({ changes, overallDateImplemented }) => {
+const ImplementedChangesList: React.FC<ImplementedChangesListProps> = ({ changes }) => {
   return (
-    <PageBlock
+    <InfoBlock
       title={'Implemented Changes'}
-      headerRight={<>{overallDateImplemented ? datePipe(overallDateImplemented) : emDashPipe('')}</>}
-    >
-      <List>
-        {changes.map((ic, idx) => (
-          <ListItem key={idx}>
-            <DynamicTooltip title={`${fullNamePipe(ic.implementer)} - ${datePipe(ic.dateImplemented)}`}>
-              {
-                <Typography>
-                  [
+      children={
```

Comment:
children are any components that are inside the tags for a parent component, so instead of this, you should do <InfoBlock> <List>...</List> </InfoBlock>